Gruntfile.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+module.exports = function (grunt) {
+
+    grunt.initConfig({
+        eslint: {
+            all: ["*.js", "src/**/*.js", "tests/**/*.js"]
+        },
+        jsonlint: {
+            all: [".eslintrc.json", "package.json", "configs/**/*.json", "src/**/*.json", "tests/**/*.json", "devpmtTestData/**/*.json"]
+        },
+        shell: {
```

Comment:
If you include any JSON5 files in the project (after the cleanup recommended), you should also add the JSON5 linting task

Reply:
The JSON5 task is part of the project now